server/src/main/java/io/druid/client/cache/Cache.java

```diff
@@ -41,6 +42,7 @@
 
   public class NamedKey
   {
+    // This is different than the dimension extraction "namespace" but the concept is very similar
```

Comment:
Just saying that they are different is not too useful and it's not clear in which way they are very similar.  Please actually explain what is different and why they are similar (or just leave out the comment)